doc-content/enterprise-only/employee-rostering/er-employees-proc.adoc

```diff
@@ -5,18 +5,18 @@ You must enter the list of all employees of the business, the skills they posses
 
 .Procedure
 
-. To enter or change the list of employees in the Optaweb employee rostering application web interface, click the *Employees* tab.
+. To enter or change the list of employees in the OptaWeb employee rostering application web interface, click the *Employees* tab.
 +
 You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other employees in the list.
 +
 You can enter any part of an employee name in the *Search* box to search for employees.
 +
 . Complete the following steps to add a new employee:
-.. Click *Add Employee*.
-.. Enter the name of the employee in the text field under *Employee name*.
+.. Click *Add*.
+.. Enter the name of the employee in the text field under *Name*.
 .. Optionally, select one or more skills from the drop-down list under *Skill set*.
 .. Select a contract from the drop-down list under *Contract*.
-.. Click *Save Employee*.
+.. Click the Save icon.
```

Comment:
Question - does this use a floppy disk icon or is it titled?

Suggestion - bold name if titled or use lowercase if not:

s/Click the Save icon./Click the save icon/Click the *Save* icon.

Reply:
It uses a floppy disk icon, with no title. A message will appear when the icon is clicked stating an employee is added/updated.